app/build.gradle

```diff
@@ -52,6 +52,14 @@ android {
             setIgnore(true)
         }
     }
+
+    splits {
+        abi {
+            enable true
+            reset()
+            include 'x86', 'armeabi-v7a'
+        }
+    }
```

Comment:
We already have the separate flavors per ABI (because there's no ALL-ABIs GeckoView yet). How does this split affect that? What APKs does the build generate?